Specs/Laser.md

```diff
@@ -0,0 +1,104 @@
+| 0:HasProperty | Variable | 2:DeviceClass | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | 2:SerialNumber | 0:String | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | **YearOfConstruction** | UInt16 | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | MonthOfConstruction | Byte | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | InitialOperationDate | DateTime | 0:PropertyType | O, RO |
+| Applied from IMachineTagNameplateType |     |     |     |     |     |
+| 0:HasProperty | Variable | 2:AssetId | 0:String | 0:PropertyType | O, RW |
+| 0:HasProperty | Variable | 2:ComponentName | 0:LocalizedText | 0:PropertyType | O, RW |
+| 0:HasProperty | Variable | **Location** | 0:String | 0:PropertyType | **M**, RW |
+
+#### Special requirements for correct location of the machine icon on the dashboard
+
+The instance _Location_ property is evaluated according to the special requirements detailed [here](../Dashboard.md#location-of-fair-machine-and-software-icons-on-the-dashboard) to place a map icon for the machine.
+
+### Adaption of the provided information models for your purpose
+
+- The provided NodeSet contains the address space with the ObjectTypes, VariableTypes and DataTypes of the Laser Systems companion specification and **must not** be changed.
+- Change the URI of the instance namespace (optional, but recommended)
+
+  `http://www.<MANUFACTURERDOMAIN>/example`
+
+- The rules for creating a URI can be found online. Please follow [this link](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier).
+In the case of a URL as a URI, it is not necessary that the site actually exists, it only identifies the machine uniquely.
+In order to simplify debugging it is recommended to choose a URI which allows an identification of the machine also for third persons, e.g. by including the company name.
+
+- The machine will get a unique identifier like clientId%3Dvdw%7Csmplsrv1%3Bnsu%3Dhttp%3A_2F_2Fwww.wenzel-group.com_2FWenzel_20LH_2087_2F%3Bi%3D66382 <clientId><mqttusername><NamespaceUri><EntryNodeId> in the MQTT tree and in the Dashboard URL.
+
+## Value mapping between OPC UA companion specification and umati.app MachineTool page
+
+### Laser System - Overview
+
+![Overview](../img/Laser/Overview.png "Laser System Overview")
+
+### Active Program Statusbar
+
+For the status overview in the dashboard, the _StateNumber_ of the _4:Monitoring_.LasterSystemStatus.LaserSystemState_StateMachineType.CurrentState.Number_ is stored and plotted over time. The status bar **does NOT relate to the stacklight** status!
+
+Each status is assigned to a color, the color scheme is here (subject to change):
+
+| State |Number | Color |
+| --- | --- | --- |
+| Off |0| &#EAEAEA; Grey |
+| EnergySaving |1| &#EAEAEA; Grey |
+| Idle |2| &#EAEAEA; Grey |
+| SetUp |3|&#EAEAEA; Grey |
+| LaserReady |4|&#EAEAEA; Grey |
+| Maintenance | 5 | &#EAEAEA; Grey|
+| Error | 6 | &#EAEAEA; Grey|
```

Comment:
All grey? If yes please fix to `&#9898; Grey` for &#9898; Grey

Reply:
that was suggest from the group